lxd/events.go

```diff
@@ -129,9 +132,12 @@ func eventSend(eventType string, eventMessage interface{}) error {
 				eventsLock.Unlock()
```

Comment:
Just above here, you lock when you already have the eventsLock (I think), which will deadlock. And there's no matching Unlock for the return below.